src/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+}
+
+.stars--0 .stars__star:nth-child(n) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(n),
+.stars--2 .stars__star:nth-child(n),
+.stars--3 .stars__star:nth-child(n),
+.stars--4 .stars__star:nth-child(n),
+.stars--5 .stars__star:nth-child(n) {
```

Comment:
do you really need to repeat 5 different selectors? try to write it with 1 selector